src/drivers/uavcan_v1/SubscriptionManager.hpp

```diff
@@ -60,6 +60,7 @@
 typedef struct {
 	const char *px4_name;
 	UavcanDynamicPortSubscriber *(*create_sub)(CanardInstance &ins, UavcanParamManager &pmgr) {};
+	UavcanDynamicPortSubscriber *instance {nullptr};
```

Comment:
@PetervdPerk-NXP I'm open to other suggestions on how to handle this, but storing a pointer to the Subscriber instance was my fix for determining whether any given Subscriber was instantiated or not in future calls to `updateDynamicSubscriptions()`.

Reply:
Hm I get it why that would be faster, but I made `_uavcan_subs const` to save RAM by storing it into flash.

Another way is to iterate over the _dynsubscribers when a parameter gets updated to figure out if the subscriber already has been created, this does take more CPU cycles but saves on RAM usage, Anyway I would argue that parameter updates shouldn't occur that often e.g. only during preflight.

Reply:
`const` structs like this get stored in flash vs. RAM?  I guess that makes sense; didn't even consider that.  Learn something every day!

I'll try to make that change later today.

Reply:
Changes made